docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md

```diff
@@ -12,51 +12,191 @@ The syntax of `MATCH` is relatively more flexible compared with that of other qu
+
+nebula> REBUILD TAG INDEX name; // Rebuild the index.
++------------+
+| New Job Id |
++------------+
+| 121        |
++------------+
+Got 1 rows  (time spent 2676/3990 us)
+
+nebula> SHOW JOB 121; // Make sure the rebuild job succeeded.
++----------------+---------------------+------------+------------+------------+
+| Job Id(TaskId) | Command(Dest)       | Status     | Start Time | Stop Time  |
++----------------+---------------------+------------+------------+------------+
+| 121            | "REBUILD_TAG_INDEX" | "FINISHED" | 1607073046 | 1607073046 |
++----------------+---------------------+------------+------------+------------+
+| 0              | "storaged2"         | "FINISHED" | 1607073046 | 1607073046 |
++----------------+---------------------+------------+------------+------------+
+| 1              | "storaged0"         | "FINISHED" | 1607073046 | 1607073046 |
++----------------+---------------------+------------+------------+------------+
+| 2              | "storaged1"         | "FINISHED" | 1607073046 | 1607073046 |
++----------------+---------------------+------------+------------+------------+
+Got 4 rows (time spent 1186/2998 us)
+```
+
+### Match vertices
+
+You can use a user-defined variable in a pair of parentheses to represent a vertex in a pattern. For example: `(v)`.
+
+Tags are specified with `:<tag_name>` and tag properties are specified with `{<prop_name>: <prop_value>}` after the `:<tag_name>`.
+
+The following example uses the name property to match a vertex.
+
+```nGQL
+nebula> MATCH (v:player{name:"Tim Duncan"}) RETURN v;
++----------------------------------------------------+
+| v                                                  |
++----------------------------------------------------+
+| ("player100" :player{name: "Tim Duncan", age: 42}) |
++----------------------------------------------------+
+```
+
+The WHERE clause can do the same thing:
+
+```nGQL
+nebula> MATCH (v:player) WHERE v.name == "Tim Duncan" RETURN v;
++----------------------------------------------------+
+| v                                                  |
++----------------------------------------------------+
+| ("player100" :player{name: "Tim Duncan", age: 42}) |
++----------------------------------------------------+
+```
+
+> **OpenCypher Compatibility**
 >
 > * In nGQL, `==` is the equality operator and `=` is the assignment operator (as in C++ or Java).
 > * In openCypher 9, `=` is the equality operator.
 
+Or you can use the VID to match a vertex. The `id()` function can retrieve the VID of a vertex.
+
+```nGQL
+nebula> MATCH (v) WHERE id(v) == 'player101' RETURN v;
++---------------------------------------------------+
+| v                                                 |
++---------------------------------------------------+
+| (player101) player.name:Tony Parker,player.age:36 |
++---------------------------------------------------+
+Got 1 rows (time spent 1710/2406 us)
+```
+
+### Match connected vertices
+
```

Comment:
nGQL 1.0 compatibality issue:
`--` is used as line comment in nGQL 1.0
in nGQL 2.0 `--` represents an undirect edge.